rewatch/src/build/parse.rs

```diff
@@ -268,6 +292,11 @@ pub fn parser_args(
     let experimental_features_args = root_config.get_experimental_features_args();
     let bsc_flags = config::flatten_flags(&package_config.compiler_flags);
 
+    // TODO: The legacy bsb build system applies warning/error flags to both parsing and compilation phases.
```

Comment:
This was a bit unexpected. `-warn-error` is passed by bsb during parsing. However, Rewatch is not passing any warn/error flags found in rescript.json. Is this intentional @jfrolich or @rolandpeelen?

Reply:
no good catch!